src/ios/CDVKeyboard.m

```diff
@@ -207,12 +212,24 @@ - (void)shrinkViewKeyboardWillChangeFrame:(NSNotification*)notif
 
 - (void)scrollViewDidScroll:(UIScrollView*)scrollView
 {
+    /*
+     Costin Moraru
+     
+     On ios 12 with default cordova webview won't scroll webview content to bottom with bounds.
+     Used setContentOffset to scroll webview content to 0.0f.
+     
+     */
+//    if (_shrinkView && _keyboardIsVisible) {
+//        CGFloat maxY = scrollView.contentSize.height - scrollView.bounds.size.height;
+//        if (scrollView.bounds.origin.y > maxY) {
+//            scrollView.bounds = CGRectMake(scrollView.bounds.origin.x, maxY,
+//                                           scrollView.bounds.size.width, scrollView.bounds.size.height);
+//        }
+//    }
     if (_shrinkView && _keyboardIsVisible) {
-        CGFloat maxY = scrollView.contentSize.height - scrollView.bounds.size.height;
-        if (scrollView.bounds.origin.y > maxY) {
-            scrollView.bounds = CGRectMake(scrollView.bounds.origin.x, maxY,
-                                           scrollView.bounds.size.width, scrollView.bounds.size.height);
-        }
+        // Scroll webview content to bottom
+        CGPoint bottomOffset = CGPointMake(0.0f, 0.0f);
+        [self.webView.scrollView setContentOffset:bottomOffset animated:NO];
```

Comment:
This works, but doesn't match expectations across the board. This scrollsToTop when the keyboard is opened. In some instances that means scrolling the input out of view.